### ISSUE TITLE
light version is missing?

### ISSUE THREAD

**Issue by zhuguangxiang** [kiliman/operator-mono-lig#69]

**Comment by waggertron** [comment]
Is there a way to process the light font as well?

**Comment by Sawtaytoes** [comment]
I have a plugin to swap between various versions of Operator Mono based on the font size. I'm also using the Light version and would love to see ligatures supported there as well!

```json
{
	"font_face_large": "Operator Mono Light",
	"font_face_medium": "Operator Mono Book",
	"font_face_small": "Operator Mono SSm Medium",

	"font_size_large_up": 19,
	"font_size_medium_up": 11,
}
```

**Comment by Sawtaytoes** [comment]
This is already talked about in #15.

**Comment by kiliman** [comment]
The new build process in `features/contextsubst` allows me to generate individual ligatures as separate glyph files. This way I don't have to generate the glyph for all font weights at the same time. You can see that I have the completed Book SSm weight, but only half the Light (regular) weight done.

**Comment by CrazyCoder** [comment]
`build.bat` file on Windows doesn't build the Light version because it strips `lig` part from the folder name and `OperatorMonoSSmLig-Light` becomes `OperatorMonoSSm-ht`, hence the original `OperatorMonoSSm-Light.otf` is never found.

Workaround is to replace `Light` with something else, then replace it back:

```
:build_font
set lig=%1
set otf=%lig:Light=zzzzz%
set ttt=%otf:Lig=%
set otf=%ttt:zzzzz=Light%
```

**Comment by kiliman** [comment]
@CrazyCoder oops! Good catch.

I'll update the build script. 

Thanks

**Comment by kiliman** [comment]
I fixed the build script. This is fixed in 843fa09a7e71bab824024a64bb40d49ab9a50a82
